pkg/collector/corechecks/network-devices/cisco-sdwan/sdwan.go

```diff
@@ -44,6 +44,7 @@ type checkCfg struct {
 	CAFile                    string `yaml:"ca_file"`
 	SendNDMMetadata           *bool  `yaml:"send_ndm_metadata"`
 	MinCollectionInterval     int    `yaml:"min_collection_interval"`
+	CollectBFDSessionStatus   bool   `yaml:"collect_bfd_session_status"`
```

Comment:
Is there an example config yaml somewhere yet?

Reply:
Next PR 😄